packages/create-hops-app/lib/package-manager.js

```diff
@@ -97,3 +97,10 @@ function isPackageInstalled(name) {
   }
 }
 module.exports.isPackageInstalled = isPackageInstalled;
+
+function isUsingNpxOrNpm(env) {
```

Comment:
What is the reason to pass the env map instead of accessing `process.env` here?

Reply:
It makes the function side-effect free and, IMHO makes it easier to understand [the callsite](https://github.com/xing/hops/pull/711/files#diff-4544895faadfb1612bf6b35db49e970dR34) because you know that this function will return something that is calculated from the env variables.